podman/api/uds.py

```diff
@@ -5,9 +5,9 @@
 from urllib.parse import unquote, urlparse
 
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3 import HTTPConnectionPool  # pylint: disable=import-error
-from requests.packages.urllib3.connection import HTTPConnection  # pylint: disable=import-error
-from requests.packages.urllib3.util import Timeout  # pylint: disable=import-error
+from urllib3 import HTTPConnectionPool
+from urllib3.connection import HTTPConnection
+from urllib3.util import Timeout
```

Comment:
would it make sense to wrap this in an `except ImportError` clause, loading `requests.packages.urllib3` stuffs, in case of an older environment?

Reply:
It is easy enough to do, and having the explicate imports make development tools much happier. Update coming...